index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- fontawesme link -->
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css">
+
+
+</head>
+<body>
+    <nav>
+        <button class="nav-btn navbars"><i class="fa-solid fa-bars fa-2x"></i></button>
+        <div class="navbar">
+            <ul class="top-navbar-menu">
+                <li><a href=""><img class="facebook-logo" src="image/facebool-logo.png" alt="facebook logo"></a></li>
+                <li><a href=""><img class="twitter-logo" src="image/twitter-logo.png" alt="twitter logo"></a></li>
+                <li><a href="">English</a></li>
+                <li><a href="">My Page</a></li>
+                <li><a href="">Log out</a></li>
+            </ul>
+            <div class="bottom-navbar">
+                <button class="nav-btn navxmark"><i style="font-size: 30px;" class="fa-solid fa-xmark fa-2x"></i></button>
+                <a class="logo" href="index.html">Logo</a>
+                <ul class="bottom-navbar-menu">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a href="">Campiagn</a></li>
+                </ul>   
+            </div>
+        </div>
+    </nav>
+    <header>
+        <div class="headline container">
+            <p class="sub-title">"Kabul Mesuem Exhibition"</p>
+            <h1 class="title">FOURTH ROUND OF KABUL MESUEN EXHABITION</h1>
+            <div class="short-info">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer interdum quis dui quis tincidunt. Praesent feugiat nibh sed lorem finibus, sit amet consectetur nisi porttitor. Vestibulum eget tincidunt quam.</p>
+            </div>
+            <p class="date">2023.5.10(FRI) ~ 12(SUN)</p>
+            <p>@ Darulman Kabul Museum, Art Center Nabi</p>
+        </div>
```

Comment:
- [ ]  As requested by the previous reviewer, kindly fix the background to match the given design, and also fix the color of the main title.

Your design
![image](https://user-images.githubusercontent.com/81932478/233794811-054a87b1-f1af-443e-88ee-ae3ba9fd77ea.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/233794791-c19c4a9f-45d1-492b-ada1-67144e3d5db9.png)

## Notice the difference between the color of your text in your design and the color of the text in the given design

Reply:
Great job, updating the text color, size, and weight to match the provided design. Almost done, but there is still an important change to work on
As request by the previous reviewer:

> * [ ]   As requested by the previous reviewer, kindly fix the background to match the given design, and also fix the color of the main title.
> 
> Your design ![image](https://user-images.githubusercontent.com/81932478/233794811-054a87b1-f1af-443e-88ee-ae3ba9fd77ea.png)
> 
> Given design ![image](https://user-images.githubusercontent.com/81932478/233794791-c19c4a9f-45d1-492b-ada1-67144e3d5db9.png)

- [ ] Please update the background of the headline (header) of the home page and about page to match the figma design. The background should have a kind of gray (transparent) layer (**tip:** you did the same thing on the submit image on the about page, with orange transparent images). 


This will make your project match the design and your texts more readable (think about users who have vision handicaps). This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪